src/main/java/se/sundsvall/casemanagement/api/CaseResource.java

```diff
@@ -81,7 +85,14 @@ public ResponseEntity<Void> putCase(
 		@RequestBody
 		@Valid CaseDTO caseDTOInput) {
 
-		if (caseDTOInput instanceof OtherCaseDTO otherCaseDTO) {
+		if (caseDTOInput instanceof ByggRCaseDTO byggRCaseDTO) {
```

Comment:
The description for this method is `Only available for cases created in CaseData.` Also it would probably be cleaner to move this to the Serviceclass instead of having the if/else logic in the resource.